src/20240707_AaveV3Ethereum_IncreaseCCIPFacilitatorCapacity/AaveV3Arbitrum_IncreaseCCIPFacilitatorCapacity_20240707.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV3ArbitrumAssets} from 'aave-address-book/AaveV3Arbitrum.sol';
+import {IProposalGenericExecutor} from 'aave-helpers/interfaces/IProposalGenericExecutor.sol';
+
+interface IGHOToken {
+  function setFacilitatorBucketCapacity(address facilitator, uint128 limit) external;
+}
+
+/**
+ * @title Increase CCIP Facilitator Capacity
+ * @author @karpatkey_TokenLogic
+ * - Snapshot: Direct-to-AIP
+ * - Discussion: https://governance.aave.com/t/arfc-gho-arb-increase-ccip-facilitator-capacity/18169
+ */
+contract AaveV3Arbitrum_IncreaseCCIPFacilitatorCapacity_20240707 is IProposalGenericExecutor {
+  address public constant FACILITATOR = 0xF168B83598516A532a85995b52504a2Fa058C068;
```

Comment:
you can use aave addresses book here too

Reply:
@efecarranza you can add it from here: https://github.com/bgd-labs/aave-address-book/blob/main/src/MiscArbitrum.sol#L31

Reply:
done

Reply:
can't see